app/routes.js

```diff
@@ -4,4 +4,6 @@ exports.init = (app) => {
   
   app.post('/users', [], user.create);
 
+  app.post('/users/sessions', user.validateLoginInput, user.signin);
```

Comment:
As I told you above, I think it would be cleaner if you call this validate inside the controller instead of here.